### PR TITLE
Migrate collector README items to docs

### PR DESCRIPTION
**Pull request by msakrejda**
- Add pg_read_file troubleshooting docs
- Drop get_stat_replication docs

### CODE REVIEW

**Comment by msakrejda** [comment]
See also https://github.com/pganalyze/collector/pull/531

File: enterprise/log-insights.mdx [hunk 1]
```diff
@@ -16,7 +16,7 @@ First of all, you need to [set up the object storage integration](/docs/enterpri
 Once you have successfully completed this:
 
 * For Amazon RDS and Aurora you can turn on the Log Insights flag on your servers in the settings page, or run the collector separately
-* For self-managed servers, you can either [add the pg_read_file helper](https://github.com/pganalyze/collector#setting-up-log-pg_read_file-helper) and then turn on the Log Insights flag on the server settings page, or run the collector separately
+* For self-managed servers, you can either [add the pg_read_file helper](/docs/install/troubleshooting/pg_read_file helper) and then turn on the Log Insights flag on the server settings page, or run the collector separately
```
Comment (keiko713) [inline enterprise/log-insights.mdx:19]: typo?
```suggestion
* For self-managed servers, you can either [add the pg_read_file helper](/docs/install/troubleshooting/pg_read_file_helper) and then turn on the Log Insights flag on the server settings page, or run the collector separately
```

File: enterprise/releases/2022-08-0.mdx [hunk 2]
```diff
@@ -35,7 +35,7 @@ backlink_title: 'pganalyze Enterprise Server: Release Changelog'
 * Support for Log Insights when monitoring self-managed servers using the collector within the pganalyze Enterprise Server container
   - To enable this functionality, make sure the "Enable Log Insights" checkbox is enabled for the server
   - This utilizes the built-in Postgres function `pg_read_file` to directly read the server log files over the Postgres connection (avoiding a separate collector install)
-  - If you are using the restricted monitoring user this requires [additional setup steps](https://github.com/pganalyze/collector#setting-up-log-pg_read_file-helper)
+  - If you are using the restricted monitoring user this requires [additional setup steps](/docs/install/troubleshooting/pg_read_file helper)
```
Comment (keiko713) [inline enterprise/releases/2022-08-0.mdx:38]: ditto

```suggestion
  - If you are using the restricted monitoring user this requires [additional setup steps](/docs/install/troubleshooting/pg_read_file_helper)
```

**Review by keiko713 — APPROVED** [review]
Small typos but looks good 👍

File: enterprise/log-insights.mdx (hunk 1 above)
Comment (msakrejda) [reply enterprise/log-insights.mdx:19]: Yeah, good catch.

**Pull request merged**